tensorflow/lite/g3doc/guide/build_cmake.md

```diff
@@ -57,11 +57,11 @@ provide `-DCMAKE_BUILD_TYPE=Debug` option.
 cmake ../tensorflow_src/tensorflow/lite -DCMAKE_BUILD_TYPE=Debug
 ```
 
-#### Build with kernel unit tests
+#### Build with kernel unit tests, evaluation tools
 
-In order to be able to run kernel tests, you need to provide
-'-DTFLITE_KERNEL_TEST=on' flag. Unit test cross-compilation specifics can be
-found in the next subsection.
+In order to build kernel tests and/or evaluation tools, you need to provide
+`-DTFLITE_KERNEL_TEST=on`, `-DTFLITE_EVAL_TOOLS=on` flags, respectively. Cross-compilation specifics
```

Comment:
I think it's better to build like benchmark_model and larbel_image without using TFLITE_KERNEL_TEST

https://www.tensorflow.org/lite/guide/build_cmake#step_6_build_tensorflow_lite_benchmark_tool_and_label_image_example_optional

Reply:
Hello @terryheo,  you meant **TFLITE_EVAL_TOOLS**, not **TFLITE_KERNEL_TEST**, right? The latter is already a part of the master branch. Under this assumption I refactored the Evaluation Tools to be built similarly to benchmark_model and label_image in the most recent commit. Please let me know if there is anything else.